lizzy/apps/senza.py

```diff
@@ -28,10 +31,17 @@ def create(self, senza_yaml: str, stack_version: str, image_version: str, parame
                 args = ['--force']
                 if disable_rollback:
                     args.append('--disable-rollback')
-                self._execute('create', *args, temp_yaml.name, stack_version, image_version, *parameters)
+
+                parameters.append(TAG_CLI_FLAG)
```

Comment:
I think `parameters.extend(['-t', 'LizzyVersion={}'.format(VERSION)]` would make it clearer that both parameters belong together.